File: 백준/Silver/4134. 다음 소수/다음 소수.py
import math, sys
input = sys.stdin.readline

def isPrimeNum(n):
    prime = 0
    for i in range(n, 2*n+1):
        check = True
        for j in range(2, int(math.sqrt(i)) + 1):
            if i % j == 0:
                check = False
                break
        if check:
            prime = i
            break
    return prime

T = int(input().strip())
for _ in range(T):
    n = int(input().strip())
    if n < 2:
        print(2)
    else:
        print(isPrimeNum(n))

File: 백준/Silver/4134. 다음 소수/다음 소수.java
import java.io.*;
import java.util.*;

public class Main {
    public static void main(String[] args) throws IOException {
        Scanner sc = new Scanner(System.in);
        int T = sc.nextInt();
        for (int i = 0; i < T; i++) {
            long n = sc.nextLong();
            long prime = 2;
            if (n > 1) {
                for (long j = n; j < 2 * n + 1; j++) {
                    boolean check = true;
                    for (long k = 2; k <= Math.sqrt((long) j); k++) {
                        if (j % k == 0) {
                            check = false;
                            break;
                        }
                    }
                    if (check) {
                        prime = j;
                        break;
                    }
                }
            }
            System.out.println(prime);
        }
        sc.close();
    }
}